py3status/module.py

```diff
@@ -789,6 +790,16 @@ def deprecation_log(item):
                 param = True
             self.allow_urgent = param
 
+            # on_error
+            param = fn(self.module_full_name, "on_error")
+            if hasattr(param, "none_setting"):
+                param = "show"
+            if param not in ON_ERROR_VALUES:
+                err = "Invalid `on_error` attribute, should be one of "
+                err += "{}. Got `{}`.".format(ON_ERROR_VALUES, param)
```

Comment:
Use `make_quotes(ON_ERROR_VALUES)`. Otherwise, it would print the literal list.

Reply:
I didn't mind that much but okay, done!